x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authz/interceptor/IndicesAliasesRequestInterceptor.java

```diff
@@ -52,23 +53,22 @@ public void intercept(RequestInfo requestInfo, AuthorizationEngine authorization
             final XPackLicenseState frozenLicenseState = licenseState.copyCurrentLicenseState();
             final AuditTrail auditTrail = auditTrailService.get();
             if (frozenLicenseState.isSecurityEnabled()) {
```

Comment:
We are using both `frozenLicenseState` here and the original `licenseState` with `MemoizedSupplier` below. Given the current change, there is no benefit of having `frozenLicenseState` anymore. It can probably just be removed.